spec/controllers/spree/admin/orders/payments/payments_controller_spec.rb

```diff
@@ -44,22 +44,6 @@
         end
       end
 
-      context "with Stripe payment where payment.process! errors out" do
-        let!(:payment_method) { create(:stripe_sca_payment_method, distributors: [shop]) }
-        before do
-          allow_any_instance_of(Spree::Payment).
-            to receive(:process_offline!).
-            and_raise(Spree::Core::GatewayError.new("Payment Gateway Error"))
-        end
-
-        it "redirects to new payment page with flash error" do
-          spree_post :create, payment: params, order_id: order.number
-
-          redirects_to_new_payment_page_with_flash_error("Payment Gateway Error")
-          expect(order.reload.payments.last.state).to eq "checkout"
-        end
-      end
```

Comment:
Why is this redundant? What's the other spec that covers this same case?

Reply:
yes, I think it should be kept.